examples/node/lib/rest/api/v2010/accounts/calls/feedbackSummary.ts

```diff
@@ -20,32 +21,49 @@ import V2010 from '../../../V2010';
  * @property { string } endDate 
  * @property { string } startDate 
  */
-export interface FeedbackSummarInstanceCreateOptions {
+export interface FeedbackSummarListInstanceInstanceCreateOptions {
     endDate: string;
     startDate: string;
 }
 
-export class FeedbackSummarListInstance {
-    protected _solution: any;
-    protected _uri: string;
+export interface FeedbackSummarListInstance {
 
 
-    constructor(protected _version: V2010, accountSid: string) {
-        this._solution = { accountSid };
-        this._uri = `/2010-04-01/Accounts/${accountSid}/Calls/FeedbackSummary.json`;
-    }
-
     /**
      * Create a FeedbackSummarInstance
      *
-     * @param { FeedbackSummarInstanceCreateOptions } params - Parameter for request
+     * @param { FeedbackSummarListInstanceInstanceCreateOptions } params - Parameter for request
```

Comment:
Not sure if `ListInstanceInstance` is the best naming convention. Also, is `FeedbackSummar` a typo?

Reply:
Fixed the duplicate `Instance` naming.

The missing `y`  on summary is it trying to singularize the resource name. There's a ticket to fix this.